.gitignore

```diff
@@ -18,3 +18,10 @@ node_modules
 
 # WebStorm files
 .idea
+
+.vscode/*
```

Comment:
Can you squash this to `.vscode/` and remove the *.json lines?

Reply:
done